packages/security/src/components/GettingStartedLayout/docs/Troubleshooting/index.mdx

```diff
@@ -0,0 +1,28 @@
+import github from './images/github.png';
+import slack from './images/slack.png';
+
+### Troubleshooting
+
+Visit our [GitHub](https://github.com/carbon-design-system/ibm-security) to view
+and submit issues, or reach out on
+[Slack](https://app.slack.com/client/T1V6D3RT7/CRV39QKP0) for additional
```

Comment:
This is pointing to the slack channel for the deprecated repo

Reply:
@jlongshore I believe this is good. lmk